sky/packages/sky/lib/src/widgets/animated_container.dart

```diff
@@ -0,0 +1,229 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:sky/animation.dart';
+import 'package:sky/src/widgets/basic.dart';
+import 'package:sky/src/widgets/framework.dart';
+
+import 'package:vector_math/vector_math_64.dart';
+
+class AnimatedBoxConstraintsValue extends AnimatedValue<BoxConstraints> {
+  AnimatedBoxConstraintsValue(BoxConstraints begin, { BoxConstraints end, Curve curve: linear })
+    : super(begin, end: end, curve: curve);
+
+  // TODO(abarth): We should lerp the BoxConstraints.
```

Comment:
you should do it in this patch, it's only gonna be 8 or so lines of code, mostly copied and pasted from another lerp :-)